.travis.yml

```diff
@@ -33,13 +33,8 @@ jobs:
     - script:
       - npm run build
       - |
-        if [ $TRAVIS_PULL_REQUEST != false ] && [ $TRAVIS_SECURE_ENV_VARS == true ]; then
-            echo "Pull request with secure environment variables, running BrowserStack tests...";
-            npm run test:bs || travis_terminate 1;
-        else
-            echo "Not a pull request and/or no secure environment variables, running headless tests...";
-            npm run test:headless || travis_terminate 1;
-        fi
+          echo "Not a pull request and/or no secure environment variables, running headless tests...";
+          npm run test:headless || travis_terminate 1;
```

Comment:
Is this change just for testing?

Reply:
Yeah I just removed some things for testing purposes so that there wouldn't be unnecessary other failures